assets/src/components/v2/exception_catcher.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ErrorInfo, ReactElement } from "react";
+import { RouteComponentProps, withRouter } from "react-router-dom";
+import { getDataset } from "Util/dataset";
+import { isRealScreen } from "Util/util";
+interface ExceptionCatcherState {
+  hasError: boolean;
+  errorMessage?: Error;
+}
```

Comment:
Is this accurate? It looks like while the component declares this as its state, it never actually sets such state up, or uses it.

Reply:
A previous solution I tried used this and I just forgot to delete it after I went with this solution. Removed now.